reg/drone/service/battery/Parameters.0.1.uavcan

```diff
@@ -1,3 +1,4 @@
+@deprecated
```

Comment:
This is okay but can you please move `@deprecated` *after* the header comment for consistency with other definitions?